src/index.ts

```diff
@@ -19,25 +24,10 @@ mongoose
   .then(() => console.log("Connected to MongoDB Atlas"))
   .catch((err) => console.error("Could not connect to MongoDB Atlas", err));
 
-// Basic route for home page
-app.get("/", (_, res: Response) => {
-  res.send("Hello World!");
-});
-
-// Echo function to display request URLs
-function echo(req: Request, res: Response) {
-  res.send(req.originalUrl);
-}
-
-// Additional routes for bathroom finder functionality
-app.get("/bathrooms", echo);
-app.get("/bathrooms/:bathroomId", echo);
-app.post("/bathrooms", echo);
-app.get("/bathrooms/:bathroomId/reviews", echo);
-app.post("/bathrooms/:bathroomId/reviews", echo);
-app.get("/bathrooms/:bathroomId/tags", echo);
-app.post("/bathrooms/:bathroomId/tags", echo);
-
+// Use the routes with their base paths
+app.use("/User", userRoutes);
+app.use("/Review", reviewRoutes);
+app.use("/Facility", facilityRoutes);
```

Comment:
according to our [class text](https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design), these should be plural

Reply:
to clarify, `/users` rather than `/User`